src/simplicial-hott/08-covariant.rzk.md

```diff
@@ -1565,3 +1565,28 @@ commuting with the contravariant lifts.
     ( contravariant-fiberwise-transformation-application
         A x y f C D is-contravariant-C ϕ v)
 ```
+
+## Two sided discrete fibrations
+
+```rzk title="RS17, Definition 8.28"
+#def is-two-sided-discrete
+  (A B : U)
+  (C : A -> B -> U)
+  : U
+  :=
+    product
+      ((a : A) -> is-covariant B (\b -> C a b))
+      ((b : B) -> is-contravariant A (\ a -> C a b))
+```
+
+```rzk title="RS17, Proposition 8.29"
+#def is-two-sided-discrete-representable-is-segal
```

Comment:
I'm not sure "representable" should be part of the name here ("hom" seems more appropriate?).
@emilyriehl might know better what's the correct terminology here.

Reply:
Yeah let's go with "hom" instead of "representable" since it's shorter. :)